numpy/core/tests/test_scalarmath.py

```diff
@@ -155,7 +155,7 @@ def test_integers_to_negative_integer_power(self):
         # 2 ** -1 perhaps generic
         base = [np.array(2, dt)[()] for dt in 'bhilqBHILQ']
         for i1, i2 in itertools.product(base, exp):
-            if i1.dtype.name != 'uint64':
+            if i1.dtype != np.uint64:
```

Comment:
Interestingly dtype equality considers `long` == `long long` if they have the same size.